src/app/services/baw-api/cms/cms.service.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpClient } from "@angular/common/http";
+import { Inject, Injectable } from "@angular/core";
+import { DomSanitizer, SafeHtml } from "@angular/platform-browser";
+import { API_ROOT } from "@helpers/app-initializer/app-initializer";
+import { stringTemplate } from "@helpers/stringTemplate/stringTemplate";
+import { Observable } from "rxjs";
+import { map } from "rxjs/operators";
+
+export enum CMS {
+  HOME = "",
```

Comment:
Add comment along the lines of:

> The workbench server has a set of well-known pages that always exist for important pages. 
> Since we always need to fetch these core pages we've defined thei slugs in-app to save time.
> These slugs changing will be considered a breaking change in the server.
>
> More dyanmic pages may exist in the future, in which case the CMS API will be used to disocver what pages are available.